src/clj/rems/db/applications.clj

```diff
@@ -413,6 +440,9 @@
    (let [form (db/get-form-for-application {:application application-id})
          _ (assert form)
          application (get-application-state application-id)
+         application (if (= :workflow/dynamic (get-in application [:workflow :type]))
+                       (dynamic/assoc-possible-commands (getx-user-id) application) ; TODO move even higher?
```

Comment:
it's a good place for this since this is where we invoke `can-review?` etc.

Reply:
The reasoning would be that it uses the `context/*user*` here implicitly. At least `user-id` could be passed here explicitly.